### PR TITLE
Feature/styled login

### PR DESCRIPTION
**Pull request by jhackett1**

### CODE REVIEW

File: config/database.yml [hunk 1]
```diff
@@ -8,14 +8,14 @@ default: &default
 development:
   <<: *default
   url: postgresql://localhost/i_have_i_need_development
-  username: dev_local_user
-  password: password
+  # username: dev_local_user
```
Comment (timwis) [inline config/database.yml]: Hm, what's this for? Seems unrelated to this feature, no?

File: app/views/passwordless/sessions/create.html.erb [hunk 2]
```diff
@@ -0,0 +1,9 @@
+<% content_for :class, "login-panel--centred"%>
+
+<%= image_tag "green-tick.svg", alt: "success" %>
+
+<h2>Check your email</h2>
+
+<p>If <strong>firstname.lastname@gmail.com</strong> exists in our records, you'll get an email with a special link.</p>
```
Comment (timwis) [inline app/views/passwordless/sessions/create.html.erb]: Did you mean to use the email address the user entered here, rather than a hardcoded gmail address? Also, while we're at it, we may want to say, "If you don't get an email, speak to _xyz_" or that "you may not have an account setup by an administrator yet."

File: app/controllers/passwordless/sessions_controller.rb [hunk 3]
```diff
@@ -0,0 +1,127 @@
+# frozen_string_literal: true
+
+require "bcrypt"
+
+module Passwordless
+  # Controller for managing Passwordless sessions
+  class SessionsController < ApplicationController
```
Comment (timwis) [inline app/controllers/passwordless/sessions_controller.rb]: Why has this file been added? I was under the impression it's imported automatically by the gem and we can keep it out of the application code?

File: app/controllers/passwordless/application_controller.rb [hunk 4]
```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+module Passwordless
+    # Base for Passwordless controllers
+    class ApplicationController < ::ApplicationController
```
Comment (timwis) [inline app/controllers/passwordless/application_controller.rb]: Same question as below - why does this bit need to be in our application code? The Passwordless should add this behind the scenes.

File: app/controllers/passwordless/sessions_controller.rb (hunk 3 above)
Comment (jhackett1) [reply app/controllers/passwordless/sessions_controller.rb]: we need to modify the render call slightly to get it to call the right layout

File: config/database.yml (hunk 1 above)
Comment (jhackett1) [reply config/database.yml]: lol, this is me making it work with my local postgres. undone

File: app/views/passwordless/sessions/create.html.erb (hunk 2 above)
Comment (jhackett1) [reply app/views/passwordless/sessions/create.html.erb]: i did some more looking and figured out how to fish out the email and show it on this screen
